Formula/lib/librsvg.rb

```diff
@@ -43,24 +46,9 @@ class Librsvg < Formula
   end
 
   def install
-    args = %w[
-      --disable-Bsymbolic
-      --enable-tools=yes
-      --enable-pixbuf-loader=yes
-      --enable-introspection=yes
-    ]
-
-    system "./configure", *args, *std_configure_args
-
-    # disable updating gdk-pixbuf cache, we will do this manually in post_install
-    # https://github.com/Homebrew/homebrew/issues/40833
-    inreplace "gdk-pixbuf-loader/Makefile",
-              "$(GDK_PIXBUF_QUERYLOADERS) > $(DESTDIR)$(gdk_pixbuf_cache_file) ;",
-              ""
-
-    system "make", "install",
-      "gdk_pixbuf_binarydir=#{lib}/gdk-pixbuf-2.0/2.10.0/loaders",
-      "gdk_pixbuf_moduledir=#{lib}/gdk-pixbuf-2.0/2.10.0/loaders"
+    system "meson", "setup", "build", "-Dpixbuf-loader=disabled", *std_meson_args
```

Comment:
`-Dpixbuf-loader=disabled` looks suspicious given that we used to pass `--enable-pixbuf-loader=yes`.

Reply:
Yea, this change looks responsible for this failure: https://github.com/Homebrew/homebrew-core/actions/runs/12585492280/job/35078161710?pr=202992#step:3:234

```
  (<unknown>:24532): Gtk-WARNING **: 16:51:08.887: Found an icon but could not load it. Most likely gdk-pixbuf does not provide SVG support.
  [2025-01-02 16:51:08.888] [gtk] [critical] 
  unhandled exception (type Glib::Error) in signal handler:
  domain: gtk-icon-theme-error-quark
  code  : 0
  what  : Icon 'cherrytree' not present in theme Adwaita
```

Reply:
sounds about right, let me find another way to fix the build issue 👍